src/repository/postDAO.js

```diff
@@ -1,5 +1,5 @@
 const { PutCommand, GetCommand, UpdateCommand, QueryCommand, ScanCommand } = require("@aws-sdk/lib-dynamodb");
-const { TableName, runCommand, flaggedIndex, CLASS_POST } = require('../utilities/dynamoUtilities');
```

Comment:
I thought I used this index, but it turns out I don't. I must've hardcoded it trying to debug issues with the command when I imported the wrong command type.  Could you add it back and update the getFlaggedPosts command to use it.